content/introduction.md

```diff
@@ -1,32 +1,65 @@
 ---
-title: "Introduction"
-metaTitle: "This is the title tag of this page"
-metaDescription: "This is the meta description"
+title: "Course Introduction"
+metaTitle: "Introduction to Blockchain Course"
+metaDescription: "This is a Blockchain course centered around critical areas of innovation and research, including: scalability, oracles, privacy, Blockchain in industry, blockchain security, exchanges, and the trade-offs present in Blockchain."
 ---
 
-Some introduction text. Lists out all the headings from h1 to h6. Easy to customise.
+*DELETE BELOW, JUST TO PROVIDE OUTLINE FOR THIS INTRO*
```

Comment:
what exactly do you mean by this?

Reply:
There's a paragraph below that before the first header that has stuff from our proposal about what I said we'd say in our intro. Use it for context or ideas then delete it later :)